content/en/docs/how-to/registry-authentication.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Authenticate with OCI-compliant registries"
+description: "How to use notation to authenticate with OCI-compliant registries"
+type: docs
+weight: 1
+---
+
+You can use `notation` to authenticate to an OCI-compliant registry. This is useful for pushing and pulling signed artifacts, as well as inspecting artifacts in the registry.
+
+
+## Use `notation login` to authenticate to an OCI-compliant registry
+
+To authenticate to an OCI-compliant registry, use the `notation login` command with your registry, username, and password. For example:
+
+```console
+notation login -u <username> -p <password> <registry>
+```
+
+**NOTE:** If `notation login` is failing, you may need to configure [Docker Credential Store](https://github.com/notaryproject/notation/blob/main/specs/registry-auth.md#credential-store) for your environment.
+
+You can also set `$NOTATION_PASSWORD` and `$NOTATION_USERNAME` environment variables to avoid passing the password on the command line. For example:
+
+```console
+export NOTATION_USERNAME=<username>
+export NOTATION_PASSWORD=<password>
+notation login <registry>
```

Comment:
```suggestion
notation login <registry> -u $NOTATION_USERNAME -p $NOTATION_PASSWORD
```

Reply:
@FeynmanZhou per the previous sentence, i thought `notation` will try to use the environment variables if not username/pass parameters are set. Is that not true?